context-monitoring/src/main/java/de/atb/context/monitoring/monitors/file/FileTripletSystemMonitor.java

```diff
@@ -60,99 +45,23 @@
  * @author scholze
  * @version $LastChangedRevision: 143 $
  */
-public class FileTripletSystemMonitor extends
-    ThreadedMonitor<Triplet<File, File, File>, IMonitoringDataModel<?, ?>>
-    implements Runnable {
-
-    protected File pathToMonitor;
-    protected Thread watchDaemon;
+public class FileTripletSystemMonitor extends AbstractFileSystemMonitor<Triplet<File, File, File>> {
 
     protected Map<String, Long> filesToDates = new HashMap<>();
```

Comment:
`filesToDates` is not used and can be removed

Reply:
[x] done